pkgs/development/compilers/flutter/patches/disable-auto-update.patch

```diff
@@ -1,16 +1,3 @@
-diff --git a/bin/internal/shared.sh b/bin/internal/shared.sh
-index 3532c23114..25dfcae4c7 100644
---- a/bin/internal/shared.sh
-+++ b/bin/internal/shared.sh
-@@ -229,8 +229,6 @@ function shared::execute() {
-     exit 1
-   fi
- 
--  upgrade_flutter 7< "$PROG_NAME"
```

Comment:
Ah, `internal.sh` is no longer used. I wonder if it's worth deleting the file entirely so that if anything _does_ try to use it it will fail in a cleaner way?

Reply:
Hmm I'm not sure. But I think deleting it is a good move. 
Though, I don't think this is really related to this PR I think. I just had to edit the patch so I could update to 3.16.7 and I noticed we are never using `internal.sh` anyway.

Reply:
In that case, could we update the patch instead for now rather than removing it? I doubt it matters too much but after #280875 everything scares me...

```
--- a/bin/internal/shared.sh
+++ b/bin/internal/shared.sh
@@ -245,7 +245,7 @@ function shared::execute() {
   # and will corrupt each others' downloads.
   #
   # SHARED_NAME itself is prepared by the caller script.
-  upgrade_flutter 7< "$SHARED_NAME"
+  # upgrade_flutter 7< "$SHARED_NAME"
 
   BIN_NAME="$(basename "$PROG_NAME")"
   case "$BIN_NAME" in
```

Reply:
Done. Each flutter version has it's own patch for that file now.